apps/backend/src/middlewares/checkFunctions/electionCheck.ts

```diff
@@ -0,0 +1,36 @@
+import type { Election, User } from '@repo/votura-validators';
+import { db } from '../../db/database.js';
+
+export async function electionExists(electionId: Election['id']): Promise<boolean> {
+  // Checks if the election with the given ID exists in the database.
+  const result = await db
+    .selectFrom('Election')
+    .select(['id'])
+    .where('id', '=', electionId)
+    .limit(1)
+    .executeTakeFirst();
```

Comment:
I am not sure ... but ain't this the same?
https://kysely-org.github.io/kysely-apidoc/interfaces/SelectQueryBuilder.html#executetakefirst
https://kysely-org.github.io/kysely-apidoc/interfaces/SelectQueryBuilder.html#limit